File: src/main/java/net/igneo/icv/entity/helmet/glacialImpasse/iceSpike/IceSpikeRenderer.java
package net.igneo.icv.entity.helmet.glacialImpasse.iceSpike;

import net.minecraft.client.renderer.entity.EntityRendererProvider;
import software.bernie.geckolib.renderer.GeoEntityRenderer;

public class IceSpikeRenderer extends GeoEntityRenderer<IceSpikeEntity> {
    public IceSpikeRenderer(EntityRendererProvider.Context renderManager) {
        super(renderManager, new IceSpikeModel());
    }
}

File: src/main/java/net/igneo/icv/entity/helmet/glacialImpasse/iceSpike/IceSpikeRenderer.kt
package net.igneo.icv.entity.helmet.glacialImpasse.iceSpike

import net.minecraft.client.renderer.entity.EntityRendererProvider
import software.bernie.geckolib.renderer.GeoEntityRenderer

class IceSpikeRenderer(renderManager: EntityRendererProvider.Context) :
    GeoEntityRenderer<IceSpikeEntity?>(renderManager, IceSpikeModel())
